tests/unittest/executor/test_await_disconnected.py

```
@@ -0,0 +1,208 @@
+        accumulate and leak memory (the original bug).
+        """
+        server = MockOpenAIServer()
+        tasks = []
+        for _ in range(100):
+            request = _make_request(disconnected=False)
+            promise = _make_promise(finished=True)
+            tasks.append(
+                asyncio.create_task(
+                    server.await_disconnected(request, promise)))
+
+        done, pending = await asyncio.wait(tasks, timeout=2.0)
```

Ruff (0.15.12)

[warning] Unpacked variable `done` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)